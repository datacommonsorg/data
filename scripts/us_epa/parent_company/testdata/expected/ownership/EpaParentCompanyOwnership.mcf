Node: dcid:EpaOrganizationOwnership
description: The ownership of an EPA Facility by an Organization in a given year.
typeOf: dcs:StatisticalVariable
populationType: dcs:EpaParentCompany
measuredProperty: owns
statType: measurementResult


Node: dcid:ownershipPercentage
typeOf: schema:Property
domainIncludes: dcs:EpaParentCompany
rangeIncludes: schema:Number
name: ownershipPercentage