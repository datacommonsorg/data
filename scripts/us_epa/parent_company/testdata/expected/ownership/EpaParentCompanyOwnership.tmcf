Node: E:EpaParentCompanyOwnership->E0
typeOf: dcs:StatVarObservation
variableMeasured: dcid:EpaOrganizationOwnership
observationDate: C:EpaParentCompanyOwnership->year
observationAbout: C:EpaParentCompanyOwnership->dcid
value: C:EpaParentCompanyOwnership->epaGhgrpFacilityId
ownershipPercentage: C:EpaParentCompanyOwnership->ownership
