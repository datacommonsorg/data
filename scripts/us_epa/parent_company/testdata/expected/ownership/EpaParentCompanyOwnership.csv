dcid,epaGhgrpFacilityId,year,ownership
EpaParentCompany/NationalFuelGasCo,epaGhgrpFacilityId/1001829,,100
EpaParentCompany/UsGovernment,epaGhgrpFacilityId/1001818,2012,100
EpaParentCompany/UsGovernment,epaGhgrpFacilityId/1001818,2014,100
EpaParentCompany/UsGovernment,epaGhgrpFacilityId/1001818,2015,100
EpaParentCompany/BrownAndProvidencePlantations,epaGhgrpFacilityId/1001811,2020,100
EpaParentCompany/ChevronUsa,epaGhgrpFacilityId/1001799,2013,1.2
EpaParentCompany/ConocophillipsAlaskaInc,epaGhgrpFacilityId/1001799,2013,36.1
EpaParentCompany/BpAmericaInc,epaGhgrpFacilityId/1001799,2013,26.4
EpaParentCompany/ExxonmobilCorp,epaGhgrpFacilityId/1001799,2013,36.4
